prow/plugins/testfreeze/testfreeze_test.go

```diff
@@ -48,9 +48,10 @@ func TestHandle(t *testing.T) {
 			branch: defaultKubernetesBranch,
 			prepare: func(mock *testfreezefakes.FakeVerifier) {
 				mock.CheckInTestFreezeReturns(&checker.Result{
-					InTestFreeze: true,
-					Tag:          "v1.23.0",
-					Branch:       "release-1.23",
+					InTestFreeze:    true,
+					Tag:             "v1.23.0",
+					Branch:          "release-1.23",
+					LastFastForward: "Wed May  4 16:15:37 CEST 2022",
```

Comment:
non-blocking and nit: this extra space between `May` and `4` is intentional or this is an output when using the function?

Reply:
It's intentional, depending on which format we want to apply to the date. The GitHub markdown parser would strip the double spaces anyhow.